.travis/publish.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly tag="${TRAVIS_TAG}"
+readonly blobaccount="${DEPLOY_BLOB_ACCOUNT_NAME}"
+readonly blobkey="${DEPLOY_BLOB_ACCOUNT_KEY}"
+readonly blobcontainer="${DEPLOY_BLOB_CONTAINER}"
+readonly blobname="fortis-${tag}.jar"
+
+log() {
+  echo "$@" >&2
+}
+
+check_preconditions() {
+  if [ -z "${tag}" ]; then
+    log "Build is not a tag, skipping publish"
+    exit 0
+  fi
+  if [ -z "${blobaccount}" ] || [ -z "${blobkey}" ] || [ -z "${blobcontainer}" ]; then
+    log "Azure blob connection is not set, unable to publish builds"
+    exit 1
+  fi
+}
+
+install_azure_cli() {
+  curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+  sudo apt-get update
+  sudo apt-get install -y -qq nodejs
+  sudo npm install -g npm
```

Comment:
Couldnt we minimize the build time by installing the `azure-cli` apt-get package as opposed to installing node? i.e. `sudo apt-get -y update && sudo apt-get install -y azure-cli`